TCP_UDP_server.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"log"
+	"bytes"
+	"fmt"
+	"io"
+	"sync"
+	"net"
+)
+var routinecount sync.WaitGroup
+var waitForgoroutine sync.WaitGroup
+var message string
+
+func main()  {
+	waitForgoroutine.Add(1)
+	go deployTcpServer()
+	deployUdpserver()
+	waitForgoroutine.Wait()
+
+}
+
+func deployTcpServer()  {
+
+	defer waitForgoroutine.Done()
+	// listens for connection
+	listens,error:=net.Listen("tcp","localhost:5002")
```

Comment:
Difference between `Listen` and `ListenTCP`?

Reply:
To listen on local network Listen works fine although ListenTcp also work fine but main difference bw both of them is that in ListenTCP network must be only TCP but in listen it can be "tcp" "tcp4" "tcp6" "unixpacket " 
https://golang.org/pkg/net/#Listen

Reply:
So we cannot pass tcp6 or tcp4 to `ListenTCP()`?

Reply:
for {

		buff := make([]byte, 1024)

		_, buffError := connect.Read(buff)

		if buffError != nil {
			log.Fatal("here it is: ", buffError)    //here is error 

		}

		
		message := string(buff[:])
		fmt.Println(message)

	}



when i use for loop in tcp client file then get EOF error

Reply:
no we can use i didn't we have listentcp so that i used listen :)

Reply:
I suggested you to used this loop in udp listener no tcp :)

Reply:
> no we can use i didn't we have listentcp so that i used listen :)
Gotcha!